webapp/src/js/directives/header.js

```diff
@@ -52,22 +54,15 @@ angular.module('inboxDirectives').directive('mmHeader', function() {
         DBSync.sync(true);
       };
 
-      const tabs = [{name:'messages',state:'messages.detail',icon:'fa-envelope',translation:'Messages',
-        permissions:['can_view_messages','can_view_messages_tab'],typeName:'message'},
-      {name:'tasks',state:'tasks.detail',icon:'fa-flag',translation:'Tasks',
-        permissions:['can_view_tasks','can_view_tasks_tab']},
-      {name:'reports',state:'reports.detail',icon:'fa-list-alt',translation:'Reports',
-        permissions:['can_view_reports','can_view_reports_tab'],typeName:'report'},
-      {name:'contacts',state:'contacts.detail',icon:'fa-user',translation:'Contacts',
-        permissions:['can_view_contacts','can_view_contacts_tab']},
-      {name:'analytics',state:'analytics',icon:'fa-bar-chart-o',translation:'Analytics',
-        permissions:['can_view_analytics','can_view_analytics_tab']}];
 
-      $q.all(tabs.map(tab => Auth.has(tab.permissions))).then(results => {
-        ctrl.permittedTabs = tabs.filter((tab,index) => results[index]);
-      });
 
-      ctrl.setMinimalTabs(tabs.length > 3);
```

Comment:
Apparently, this never worked: https://medic.slack.com/archives/C024KTGRW/p1591305870027500